### PR TITLE
FIX: Missed deprecation warning

### PR DESCRIPTION
**Pull request by maxcapodi78**
new_desktop_session deprecation warning was missed in App initialization

### CODE REVIEW

**Review by Samuelopez-ansys — APPROVED** [review]

**Review by Samuelopez-ansys — APPROVED** [review]

**Review by SMoraisAnsys — APPROVED** [review]
LGTM, thanks for this fix !

File: .github/workflows/ci_cd.yml [hunk 1]
```diff
@@ -458,7 +458,7 @@ jobs:
           command: |
             export LD_LIBRARY_PATH=${{ env.ANSYSEM_ROOT241 }}/common/mono/Linux64/lib64:${{ env.ANSYSEM_ROOT241 }}/Delcross:$LD_LIBRARY_PATH
             source .venv/bin/activate
-            pytest -n 4 --dist loadfile --durations=50 -v --cov=pyaedt --cov-report=xml --cov-report=html --junitxml=junit/test-results.xml _unittest
+            pytest -n 2 --dist loadfile --durations=50 -v --cov=pyaedt --cov-report=xml --cov-report=html --junitxml=junit/test-results.xml _unittest
```
Comment (MaxJPRey) [inline .github/workflows/ci_cd.yml:461]: @maxcapodi78 Do we have issue on some VMs when running the tests with 4 CPUs?

Comment (Samuelopez-ansys) [reply .github/workflows/ci_cd.yml:461]: @MaxJPRey We faced some random fails in Linux in the last PRs, we think this could be the issue.

Comment (MaxJPRey) [reply .github/workflows/ci_cd.yml:461]: OK, good to know. Thanks @Samuelopez-ansys !

**Review by MaxJPRey — APPROVED** [review]

**Pull request merged**